H_w 2/task_8.rb

```diff
@@ -0,0 +1,7 @@
+#8. There is a hash where keys and values are strings. Modify it: all keys should be symbols and all values should be integers. If you couldn't modify some value set it to nil
```

Comment:
" If you couldn't modify some value set it to nil"

can't see any code to implement this logic